File: LeetCode/1-100/04. 寻找两个正序数组的中位数.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number}
 */
var findMedianSortedArrays = function (nums1, nums2) {
  const arr = [...nums1, ...nums2]
  arr.sort((a, b) => a - b)
  let length = arr.length
  let index = Math.floor(arr.length / 2)
  if (length % 2 === 0) {
    return (arr[index - 1] + arr[index]) / 2
  } else {
    return arr[index]
  }
}

File: LeetCode/1-100/04. 寻找两个正序数组的中位数.go
func findMedianSortedArrays(nums1 []int, nums2 []int) float64 {
	n := len(nums1) + len(nums2)
	var num3 []int
	num3 = append(num3, nums1...)
	num3 = append(num3, nums2...)
	sort.Ints(num3)
	mid := len(num3) / 2
	if n%2 == 0 {
		return float64(num3[mid] + num3[mid-1]) / 2
	} else {
		return float64(num3[mid])
	}
}